src/encoded/schemas/atac_aggregate_metrics.json

```diff
@@ -0,0 +1,95 @@
+{
+	"title": "Atac aggregate metrics",
+	"description": "Schema for submitting metadata for aggregate ATAC-seq metrics",
+	"id": "/profiles/atac_summary_metrics.json",
+	"schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"required": ["quality_metric_of"],
+	"identifyingProperties": ["uuid"],
+	"additionalProperties": false,
+	"mixinProperties": [
+        {"$ref": "mixins.json#/aliases"},
+        {"$ref": "mixins.json#/documents"},
+        {"$ref": "mixins.json#/notes"},
+        {"$ref": "mixins.json#/submitted"},
+        {"$ref": "mixins.json#/schema_version"},
+        {"$ref": "mixins.json#/uuid"},
+        {"$ref": "mixins.json#/standard_status"},
+        {"$ref": "mixins.json#/quality_metric_of"}
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "annotated_cells": {
+        	"title": "Annotated cells",
+        	"description": "Estimated number of cells.",
+        	"type": "integer"
+        },
+        "frac_cut_fragments_in_peaks": {
+        	"title": "Frac cut gragments in peaks",
```

Comment:
I think we should make the titles as human-friendly as possible, so write out 'Fraction'

Reply:
Write out the complete work for just the titles, and keep the field name matching the output summary.csv?

Reply:
I don't have a strong opinion about the field name, so long as all field names are consistent.